### ISSUE TITLE
Commissieleden pagina automatisch laten updaten

### ISSUE THREAD

**Issue by indiePeeters** [esac-ic/esac.nl#165]
na dat #162 mogelijk is, kunnen de commissieleden om de commissiepagina automatisch geupdate worden.

**Comment by wouterbles** [comment]
Dit is eigenlijk nog best complex. Want je wil ook nog makkelijk de foto's en beschrijvingen van commissies aanpassen.

**Comment by indiePeeters** [comment]
Nee, de beschrijving en foto's ( wat eigenlijk naar mijn idee leuke foto's moeten worden wat de commissie ongeveer doet) veranderen niet snel.

De commissieleden integendeel veranderen vaker en is daarom handiger om dit te laten updaten zodat dit niet meer handmatig gedaan hoeft te worden
